lib/ui/common/image_picker.dart

```diff
@@ -0,0 +1,65 @@
+import 'dart:io';
+
+import 'package:flutter/material.dart';
+import 'package:image_cropper/image_cropper.dart';
+import 'package:image_picker/image_picker.dart' as ImagePickerLib
+    show ImagePicker, ImageSource;
+import 'package:intl/intl.dart';
+
+class _Constants {
+  static final EdgeInsets cropAreaPadding = const EdgeInsets.only(top: 20.0);
+  static final Color cropStatusBarColor = const Color(0xffffffff);
+  static final Color cropToolbarBarColor = const Color(0xffffffff);
+  static const double defaultImageRatioX = 1.0;
+  static const double defaultImageRatioY = 1.0;
+}
+
+class _Strings {
+  static final editPhoto = 'Edit Photo';
+  static final noImagePickedError = 'No image picked';
+  static final noCroppedImageError = 'No cropped image';
+}
+
+class ImagePicker {
+  static Future<bool> pickImage({
+    @required Function(File) onSuccess,
+    @required Function(String) onError,
+    @required ImagePickerLib.ImageSource imageSource,
+    @required int imageMaxHeight,
+    @required int imageMaxWidth,
+    bool circleShapeOnCrop = true,
+    double imageRatioX = _Constants.defaultImageRatioX,
+    double imageRatioY = _Constants.defaultImageRatioY,
+  }) async {
+    final file =
+        await ImagePickerLib.ImagePicker.pickImage(source: imageSource);
+
+    if (file == null) {
+      onError(_Strings.noImagePickedError);
```

Comment:
Not sure about this comment, I've seen in another PR's that text is wrapped with:

```
Intl.message(_Strings.noImagePickedError)
```

To be able to be translated to other languages, that should be done in this way?

Reply:
yep you're right

Reply:
done